test/release/examples/primers/arrayVectorOps.chpl

```diff
@@ -1,5 +1,18 @@
 // Array Vector Operations
 
+//
+// **DEPRECATION WARNING**
```

Comment:
A `.. warning::` block would put this text in a red box, if that's preferable. This works too though.

Reply:
That's definitely preferable.